docs/source/configuration/environmentvariables.md

```diff
@@ -92,10 +92,10 @@ It will enable the log of missing i18n messages (in console).
 
 ### Add add-ons via environment variable
 
-Quite often, you need different configurations and enabling components (for example, testing purposes).
-You can use the `ADDONS` environment variable to define them.
-They sum up to the ones set in `package.json` or programatically in `volto.config.js`.
+You can use the `ADDONS` environment variable to enable and configure add-ons in your project.
+They sum up to the ones set fixed in `package.json` or programatically in `volto.config.js`.
 You can specify released (published) packages (installed previously in your environment, and present already in `node_modules`), or `packages` folder local to your project (not in development, but optional, for example, Volto's testing packages).
```

Comment:
This sentence still doesn't make sense to me. Perhaps breaking it into smaller sentences will help me comprehend what you want to say?

Reply:
I'll try.